docs/user-guide/kubectl-overview.md

```diff
@@ -33,160 +33,271 @@ Documentation for other releases can be found at
+`cluster-info`	| `kubectl cluster-info` | Display endpoint information about the master and services in the cluster.
+`config`		| `kubectl config SUBCOMMAND` | Modifies kubeconfig files. See the sub-commands for details.
+`create`		| `kubectl create -f FILENAME` | Create a resource using a file or stdin.
+`delete`		| `kubectl delete ([-f FILENAME] | TYPE [(NAME | -l label | --all)])` | Delete resources either by filenames, label selectors, names, resource selectors, resources, or stdin.
+`describe`	| `kubectl describe (-f FILENAME | TYPE [NAME_PREFIX | -l label] | TYPE/NAME)` | Display all the details for one or more resources.
+`edit`		| `kubectl edit (RESOURCE/NAME | -f FILENAME)` | Edit the definition of a resource on the server.
+`exec`		| `kubectl exec POD [-c CONTAINER] -- COMMAND [args...]` | Execute a command against a container in a pod.
+`expose`		| `kubectl expose (-f FILENAME | TYPE NAME) [--port=port] [--protocol=TCP|UDP] [--target-port=number-or-name] [--name=name] [----external-ip=external-ip-of-service] [--type=type]` | Duplicate and expose a replication controller, service, or pod as a new Kubernetes service.
+`get`		| `kubectl get [(-o|--output=)json|yaml|wide|go-template=...|go-template-file=...|jsonpath=...|jsonpath-file=...] (TYPE [NAME | -l label] | TYPE/NAME ...) [flags]` | List one or more resources.
+`label`		| `kubectl label [--overwrite] (-f FILENAME | TYPE NAME) KEY_1=VAL_1 ... KEY_N=VAL_N [--resource-version=version]` | Add or update the labels of a resource.
+`logs`		| `kubectl logs [-f] [-p] POD [-c CONTAINER]` | Print the logs for a container in a pod.
+`namespace`	| `kubectl namespace [namespace]` | SUPERSEDED: Set and view the current Kubernetes namespace.
+`patch`		| `kubectl patch (-f FILENAME | TYPE NAME) -p PATCH` | Update the field(s) of a resource using stdin.
+`port-forward`	| `kubectl port-forward POD [LOCAL_PORT:]REMOTE_PORT [...[LOCAL_PORT_N:]REMOTE_PORT_N]` | Forward one or more local ports to a pod.
+`proxy`		| `kubectl proxy [--port=PORT] [--www=static-dir] [--www-prefix=prefix] [--api-prefix=prefix]` | Run a proxy to the Kubernetes API server.
+`replace`		| `kubectl replace -f FILENAME` | Replace a resource using a file or stdin.
+`rolling-update`	| `kubectl rolling-update OLD_CONTROLLER_NAME ([NEW_CONTROLLER_NAME] --image=NEW_CONTAINER_IMAGE | -f NEW_CONTROLLER_SPEC)` | Perform a rolling update using the specified replication controller.
+`run`		| `kubectl run NAME --image=image [--env="key=value"] [--port=port] [--replicas=replicas] [--dry-run=bool] [--overrides=inline-json]` | Run a specified image on the cluster.
+`scale`		| `kubectl scale [--resource-version=version] [--current-replicas=count] --replicas=COUNT (-f FILENAME | TYPE NAME)` | Set or update the size of the specified replication controller.
+`stop`		| `kubectl stop (-f FILENAME | TYPE (NAME | -l label | --all))` | Deprecated: Gracefully shut down a resource by name or filename
```

Comment:
Commented elsewhere, but deprecation is mentioned in HEAD:
https://github.com/kubernetes/kubernetes/blob/master/docs/user-guide/kubectl/kubectl_stop.md

It does still work. We just want people to stop using it so that we can remove it in the future.